tailwind.config.js

```diff
@@ -3,10 +3,19 @@ module.exports = {
     "./src/pages/*/*.{js,ts,jsx,tsx}",
     "./src/components/*/*.{js,ts,jsx,tsx}",
     "./src/services/*/*.{js,ts,jsx,tsx}",
-    "./src/utils/*/*.{js,ts,jsx,tsx}"
+    "./src/utils/*/*.{js,ts,jsx,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      backgroundImage: {
+        "users-icon": "url('../assets/icons/users-icon.svg')",
```

Comment:
if we want to add more icons or images... will we have to add them all here?

Reply:
Only if you will call them as background images